Automattic/jetpack
# Adds Social Icons from Social Logos Repo

Fixes #13765
Fixes #10663 

#### Changes proposed in this Pull Request:

* Adds the following [social icons to bring parity](https://github.com/Automattic/social-logos/issues/92) with the [Social Logos](https://github.com/Automattic/social-logos) repo (for supported themes):
* - Blogger
* - Eventbrite
* - Ghost
* - TikTok
* - Whatsapp
* - WooCommerce
* - Xanga

#### Jetpack product discussion


N/a

#### Does this pull request change what data or activity we track or use?


Nope, don't think so.

#### Testing instructions:






Testing the Social Icons Widget:

* Open the customizer
* Add a social icons widget 
* Add urls for the new icons in question, e.g.:
* - blogger.com, blogspot.com
* - eventbrite.com
* - ghost.org
* - tiktok.com
* - woocommerce.com
* - xanga.com
* You should see the logos appear in the widget area

![Screen Shot 2021-01-27 at 13 19 50](https://user-images.githubusercontent.com/33553323/106035692-67f4d880-60a2-11eb-806f-a032093800df.png)

Testing the Social Menu

* Activate a theme that supports Jetpack social icons (e.g. Dara).
* Open the customizer and add/edit the social menu.
* Add urls for the new icons in question
* You should see the logos appear in the social media menu.

![social menu change](https://user-images.githubusercontent.com/33553323/106035863-98d50d80-60a2-11eb-9aed-01c2fb71a77f.png)

#### Proposed changelog entry for your changes:


* Social Icons Widget: Adds Blogger, Eventbrite, Ghost, Tiktok, WhatsApp, WooCommerce and Xanga icons for social icons widget and social menus utilizing Jetpack's social icons.
